resources/views/components/input/shortcode-textarea.blade.php

```diff
@@ -0,0 +1,53 @@
+@props([
+    'name' => 'body',
+    'enabled' => true,
+    'watermark' => '',
+    'maxLength' => 60000,
+    'initialValue' => '',
```

Comment:
nit: It may be more ergonomic to delete this prop and use `{{ $slot }}` instead for the initial value, just like the native `<textarea>`. I believe this would lower the learning curve of this component.

```php
<textarea ...>{{ $slot }}</textarea>
```

```php
<x-input.shortcode-textarea ...>Here's my initial content!</x-input.shortcode-textarea>
```

Reply:
done